pkg/processor/runtime/python/factory.go

```diff
@@ -24,7 +24,9 @@ import (
 	"github.com/spf13/viper"
 )
 
-type factory struct{}
+type factory struct {
+	idGen *IDGenerator
```

Comment:
We have `vendor/rs/xid` for this. Do you see any advantage to not using it?

Reply:
Thought about UUIDs, but I figured a running sequence might be easier to reason about. Can change if you prefer.

Reply:
On second thought, less code is better - changing :)